lib/reviewResource.dart

```diff
@@ -133,6 +135,18 @@ class _ReviewResourceState extends State<ReviewResource> {
   int? trustworthyRating = 0;
   int? currentRating = 0;
   int? languageRating = 0;
+  String? avoidRacism = 'Yes';
+  String? avoidStereotyping = 'Yes';
+  String? avoidAppropriation = 'Yes';
+  String? avoidLanguage = 'Yes';
+  String? appropriate = 'Yes';
+  String? avoidCond = 'Yes';
+  String? avoidAgeism = 'Yes';
+  String? avoidSexism = 'Yes';

```

Comment:
Should we have these preliminary questions default to not filled in, rather than Yes? That way the user can't skip past the section.